Concentration/Card.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  Card.swift
+//  Concentration
+//
+//  Created by Oren Dinur on 08/02/2022.
+//
+
+import Foundation
+
+struct Card
+{
```

Comment:
Please follow this style:

```suggestion
struct Card {
```